### DISCUSSION TITLE
Rendering LaTeX in wikis of GitHub repos

### DISCUSSION THREAD

**Discussion by UlyssesZh** [community/community#17169] [Other Feature Feedback, Questions, & Ideas]
It seems that LaTeX codes enclosed by dollar signs are not rendered in wikis.

**Comment by ayala-io** [comment]
Yeah I brought this up in https://github.com/github/feedback/discussions/17157 as well. Maybe we should merge discussions.

**Reply by UlyssesZh** [reply]
Sorry I didn't notice (because your discussion doesn't contain the word 'LaTeX' and I didn't find it when I searched for similar problems).
